app/models/classification.rb

```diff
@@ -529,7 +529,11 @@ def find_tag
 
   def save_tag
     tag_name = Classification.name2tag(name, parent_id, ns)
-    self.tag = Tag.in_region(region_id).find_or_create_by(:name => tag_name)
+    if tag.present?
+      tag.update_attributes(:name => tag_name)
```

Comment:
Something about updating this without needing to strikes me as funny.
<substitution>
      tag.update_attributes(:name => tag_name) if tag.name != tag_name
</substitution>

Reply:
nice, let me try

Reply:
works! @kbrock 👏